Update the docs to reflect default callback URL regex issue fix (#3920)
wso2/docs-is: en/docs/deploy/security/product-level-security-guidelines.md

    
## Callback URL Regular Expressions

> Note that if you are using these scenarios, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.

Reviewer: ```suggestion
Note that if you are using these scenarios, we highly recommend you define the regular expression that validates and only allows access to specific callback URLs.
```
